### PR TITLE
Fix integration test

### PR DESCRIPTION
**Pull request by Veldhoen**
Fixed bug in S3 upload of archive

### CODE REVIEW

**Comment by Veldhoen** [comment]
@jblom unrelated to this specific PR, but it seems like the gh action uses a different Flake8 which complains about syntax within strings. Any clue as to how to solve this?? It doesn't happen locally, although I think the same Flake8 should be installed from the pyproject.toml

**Comment by Veldhoen** [comment]
Test also fails in GH actions because it can't write to ./data

File: io_util.py [hunk 1]
```diff
@@ -295,7 +295,9 @@ def obtain_input_file(s3_uri: str) -> ThisWorkerInput:
     )
     success = s3.download_file(bucket, object_name, output_folder)
     if success:
-        # TODO uncompress the <input_base>.tar.gz
+        if input_file_path.find(".tar.gz") != -1:
+            # TODO: more elegant solution for hardcoded file name
+            input_file_path = untar_input_file(input_file_path) + f"/{source_id}.input"
```
Comment (greenw0lf) [inline io_util.py:300]:
```suggestion
            input_file_path = untar_input_file(input_file_path)
```

File: io_util.py [hunk 2]
```diff
@@ -333,3 +335,4 @@ def untar_input_file(tar_file_path: str):
     path = str(Path(tar_file_path).parent)
     with tarfile.open(tar_file_path) as tar:
         tar.extractall(path=path, filter="data")  # type: ignore
+    return path
```
Comment (greenw0lf) [inline io_util.py:338]:
```suggestion
        filename = tar.getmembers()[0].name
    return path + f"/{filename}"
```

**Review by greenw0lf — COMMENTED** [review]
It runs successfully indeed! I would suggest though to also have a look at the PR I just opened for the audio extraction worker: https://github.com/beeldengeluid/dane-audio-extraction-worker/pull/16

I did some things differently, including the changes you made. Maybe it is more suitable to do it my way rather than this way? With your method, I would still have the test fail because of the file/folder names being mismatched.

**Comment by Veldhoen** [comment]
> Test also fails in GH actions because it can't write to ./data

Fixed this by using specific config-test.yml

**Pull request merged**